src/annotator/index.js

```diff
@@ -66,11 +66,16 @@ function init() {
 
   config.pluginClasses = pluginClasses;
 
-  const annotator = new Klass(document.body, config);
+  const guest = new Guest(document.body, config);
+  const sidebar = SidebarClass
+    ? new SidebarClass(document.body, config, guest)
```

Comment:
I would probably invert the last two arguments, making `config` the last one and optional.

Reply:
Yes, I think that makes sense. If its easy to do I'll make a follow-up commit in this PR. If its more complex I might separate that out into a follow-up PR.